src/services/productService.js

```diff
@@ -19,17 +19,21 @@ export function getProducts() {
     });
 }
 
-export function addProduct(productData) {
+export async function addProduct(productData) {
```

Comment:
Why do we need to make this function `async`?

Reply:
We dont. I forgot to remove this.